### ISSUE TITLE
import vdj break when clonotype id is NA

### ISSUE THREAD

**Issue by aurelBZH** [rnabioco/djvdj#62]
Hi,
Thanks for your great work on djvdj.
I had some problem with the import function. When the clonotype id is NA the function don't concatenate the row, the function can't use barcode as metadata and break. 

thanks a lot

**Comment by sheridar** [comment]
Thanks @aurelBZH, I was able to replicate the issue and will work on fixing this

**Comment by sheridar** [comment]
I adjusted the import function so it will ignore contigs that do not have a clonotype id. I have only encountered this issue in a dataset I started analyzing yesterday. In my recent dataset I found a contig that was classified as high_confidence, productive, full_length, and had the same CDR3 sequence as another contig assigned to the same cell. But this contig was not assigned a clonotype id.

It's not clear to me why this would occur, so I reached out to the 10x support staff to see if they could provide any insight.

If you re-install, the import function should now work, but the contigs that lack a clonotype id will be excluded from the data. Let me know if you have any questions/concerns or if you experience any further issues
